bin/modules/validateAddPackages.js

```diff
@@ -7,14 +7,34 @@
 const fs = require("fs");
 const removeAll = require("../utils/removeAll");
 
+const isJsonString = function (str) {
```

Comment:
@himynameisdave Nice! I've sent you an invite to a gitter room. Check the email associated with your github account, you should've got the invite there.